experimental/dds/tree2/src/feature-libraries/flex-tree/lazyEntity.ts

```diff
@@ -85,6 +91,7 @@ export abstract class LazyEntity<TSchema = unknown, TAnchor = unknown>
 	}
 
 	public abstract [boxedIterator](): IterableIterator<FlexTreeEntity>;
+	public abstract get [flexTreeMarker](): FlexTreeEntityKind;
```

Comment:
Any dev-facing docs that would be useful here?

Reply:
This should be documented on the interface not here. That said, it was not documented properly there, so I have fixed it.